NEWS.md

```diff
@@ -0,0 +1,38 @@
+# Moonlight2R 0.99.0
+
+## Summary
+
+* first release of Moonlight2R.
+
+## New features (respect to MoonlightR)
+
+* `DMA`			Driver mutation analysis
+
+* `plotDMA`		Creates one or more heatmap of the output from DMA
+
+* `plotMoonlight`	Creates heatmap of Moonlight Gene Z-scores for selected genes
+
+* `EncodePromoters`	Experimentially verified promoter sites
+
+* `LOC_protein`		Level of consequence protein
+
+* `LOC_translation`	Level of consequence translation
+
+* `LOC_transcription`	Level of consequence transcription
+
+* `NCG`			Network of Cancer Genes 7.0
+
+* `dataPRA`		output from PRA function
+
+* `dataMAF`		Mutation data from TCGA-LUAD
+
+* `dataDMA`		Output from DMA function
+
+* `cscape_somatic_output`	Cscape-somatic annotations of TCGA-LUAD 
+
+* `DEG_Mutations_Annotations`	Differentially expressed genes's Mutations 
+
+* `Oncogenic_mediators_mutation_summary`	Oncogenic Mediators Mutation Summary  
+
+* `moonlight2R`		Function to run moonlight pipeline (renamed from first version of MoonlightR)

```

Comment:
The function to run moonlight pipeline is called "moonlight" and not "moonlight2R" and it's not renamed from the first version of MoonlightR, i.e. it was also called "moonlight" in the first version of MoonlightR (not sure why it was written that it's called "moonlight2R" in the previous NEWS file)